content/200-concepts/100-components/database-drivers.mdx

```diff
@@ -17,15 +17,21 @@ Prisma Client can connect and run queries against your database using JavaScript
 
 Prisma will use the Query Engine to transform the Prisma Client query to SQL and run the generated SQL queries via the JavaScript database driver.
 
-![Query flow from the user application to the database using Prisma Client and driver adapters](./images//drivers/qe-query-engine-adapter.png)
+![Query flow from the user application to the database using Prisma Client and driver adapters](./images/drivers/qe-query-engine-adapter.png)
 
 ### Serverless driver adapters
 
+Serverless driver adapters allow you to connect to your database using other protocols, such as HTTP and WebSockets. Prisma maintains the following serverless driver adapters:
+
 - [Neon](/guides/database/neon#how-to-use-the-neon-serverless-driver-with-prisma-preview)
 - [PlanetScale](/guides/database/planetscale#how-to-use-the-planetscale-serverless-driver-with-prisma-preview)
 
 ### Database driver adapters
 
+Database driver adapters enable you to connect to your database using the default protocol that a database provider offers. For example, Turso only supports HTTP connections.
```

Comment:
I think the database driver adapters are less about the protocol, but more about that we just want to support these Node based drivers.

In the future we will also list `pg` and `Postgre.js` here for example.